src/Fable.Transforms/Python/Fable2Python.fs

```diff
@@ -4291,7 +4291,8 @@ module Util =
         // printfn "getIdentForImport: %A" (moduleName, name)
         match name with
         | None -> Path.GetFileNameWithoutExtension(moduleName)
-        | Some name -> name |> Naming.toSnakeCase |> getUniqueNameInRootScope ctx
+        | Some name -> name |> Naming.toSnakeCase
```

Comment:
This looks like just a formatting change. How can this fix anything?

Reply:
It seems like now `getUniqueNameInRootScope ctx` is applied to both branches and not just `| Some name`

What I mean, is that it is now applied to `Path.GetFileNameWithoutExtension(moduleName)` too

Reply:
Aha, now I get it 🙈 Thanks

Reply:
Correct!